roles/pdf_gen/templates/opt/tempCleaner.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+""" 2018-02-26
+
+    tempCleaner.py
+
+    Author: Scott Batchelder
+
+    This tool is designed to clean out the /tmp/ directory on servers that
+    use that space, but do not clean up after themselves(pdf_gen). This
+    tool will only delete directories that have the pattern"/tmp/tmp*/"
+    and that are older than 7 days.  An entry for this tool should be added
+    to the 'crontab' so that it runs once per day.
+
+    Make sure to execute as the proper user:
+       % sudo -u www-data "./tempCleaner.py
+
+    version 1
+"""
+
+import os
+import time
+import shutil
+import re
+
+base = r"/tmp"
+now = time.time()
+
+for dirpath, dirnames, files in os.walk(base):
+    for dirname in dirnames:
+        if "tmp" in dirname.lower():
+            f = os.path.join(dirpath, dirname)
+            if os.stat(f).st_mtime < now - 7*86400:
```

Comment:
Let's not have magic numbers. This should be a config set at the top, or passed in as a param

Reply:
or use Python's `datetime` / `timedelta`